portal/templates/initial_queries_macros.html

```diff
@@ -8,14 +9,26 @@ <h4 class="tnth-headline">{{ _("Terms of Use") }}</h4>
             <div id="termsText">
               {{ terms['asset']|safe if terms}}
             </div>
-            <input type="hidden" id="termsURL" data-url="{{ terms['agreement_url']|safe if terms}}" value=""/>
             <div id="termsExpand" class="tnth-hide">
               <a href=""><i class="fa fa-plus"></i> {{ _("View TrueNTH Terms") }}</a>
             </div>
             <div id="termsCheckbox">
-              <label id="agreeLabel" data-agree="false" data-type="terms">
-                <i class="fa fa-square-o fa-lg"></i> <span>{{ _("I consent and agree to the above terms.") }}</span>
-              </label>
+               <div class="terms-checkbox-container">
+               <!-- note these checkboxes are selectively hidden using css, as some are applicable only in Truenth and some only applicable in Eproms-->
+                <label class="custom-tou tnth-hide" data-agree="false" data-type="terms" data-tou-type="subject website consent" data-core-data-type="subject_website_consent" data-url="{{terms['url'] if terms}}">
+                  <i class="fa fa-square-o fa-lg" class="edit-view"></i> <span class="edit-view">{{ _("I have read the information notice above and consent and agree to the processing of my personal information (including my health information) on the terms described in this Consent and Terms and Conditions.") }}</span>
+                  <div class="display-view text-warning tnth-hide">{{_("You have previously provided your consent to the website during a visit at your treating site. If you would like a copy of this please contact your study contact.")}}<br/></div>
+                </label>
+                <br/>
+                <label class="general-tou" data-agree="false" data-type="terms" data-tou-type="website terms of use" data-core-data-type="website_terms_of_use" data-url="{{terms['url'] if terms}}">
+                    <i class="fa fa-square-o fa-lg"></i>&nbsp;<span class="default-tou-text">{{ _("I consent and agree to the above terms.") }}</span>
+                    <span class="custom-tou-text">{{ _("I have read the website ")}}<a href="{{url_for('portal.privacy')}}" target="_blank" class="required-link" data-core-data-subtype="privacy_policy" data-tou-type="privacy policy">{{_('privacy policy')}}</a>{{_(' and ')}}<a href="{{url_for('portal.terms_and_conditions')}}" target="_blank" class="required-link">{{_("terms") }}</a></span>
```

Comment:
Whoops, that was from Django. [Jinja ](http://jinja.pocoo.org/docs/2.9/templates/#i18n)has a similar tag with a little different syntax